RequestManagementSystem/DB/RequestDB.py

```diff
@@ -249,7 +236,8 @@ def getTableMeta():
 
   def getTables(self):
     """ Return the table names """
-    return S_OK( metadata.tables.keys() )
+    inspector = Inspector.from_engine( self.engine )
+    return inspector.get_table_names()
```

Comment:
I tried out starting with an empty DB, and it was not creating the tables. This one works (suggested by Sergio). I don't know enough of sqlAlchemy to comment properly why, nor what's better.